docker/install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bullseye/bin/sh
+
+export PATH=/bullseye/usr/bin:/bullseye/bin:/bullseye/sbin
+
+export LD_LIBRARY_PATH=/bullseye/lib/x86_64-linux-gnu:/bullseye/usr/lib/x86_64-linux-gnu/:/bullseye/lib64
+
+# Install system libraries required for java
+cp /java/lib/x86_64-linux-gnu/libz.so* /java/lib/x86_64-linux-gnu/libgcc_s.so* /lib/x86_64-linux-gnu/
+cp /java/usr/lib/x86_64-linux-gnu/libstdc++.so* /usr/lib/x86_64-linux-gnu/
+
+# Install java
+cp -r /java/etc/java-11-openjdk /etc/
+cp -r /java/etc/ssl/certs/java /etc/ssl/certs/
+cp -r /java/usr/lib/jvm /usr/lib/
+
+# Remove libs and executables from the base image we don't want
+rm /usr/lib/x86_64-linux-gnu/libssl.so*
```

Comment:
What if we add a Scala dependency that is based on this lib ?

Reply:
Do you know of any scala or java dependency that depends on this lib?  Most of the time, scala/java dependencies make no assumptions about what system libs are installed on the host.  There are a few exceptions which depend directly on system libraries (e.g. the lzo serializer we use in s3 loader) but they are rare.

And furthermore... if we do ever need it then we amend this line.  It will be a new requirement of our base image, so we adjust the libraries to meet the new requirement.

Reply:
> Do you know of any scala or java dependency that depends on this lib?

I don't, and honestly it's quite obscure to me how to know if a certain library that we use requires a `.so` under the hood or not.